dev/com.ibm.ws.grpc_fat/fat/src/com/ibm/ws/fat/grpc/ClientConfigTests.java

```diff
@@ -0,0 +1,592 @@
+            }
+        }
+    }
+
+    /**
+     * test an invalid setting, verify error CWWKG0075E occurs
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testInvalidMaxInboundMessageSize() throws Exception {
+        LOG.info("ServiceConfigTests : testInvalidMaxInboundMessageSize() : test an invalid setting.");
+
+        setServerConfiguration(GrpcClientOnly, GRPCTARGET_MSGSIZEINVALID);
+        GrpcClientOnly.waitForConfigUpdateInLogUsingMark(appName);
+        assertNotNull(GrpcClientOnly.waitForStringInLog("CWWKG0075E: The value junk is not valid for attribute maxInboundMessageSize of configuration element grpcTarget"));
+    }
+
+    /**
+     * test a very small setting, send a gRPC message
+     * exceeding the value, and check the server error
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testSmallMaxInboundMessageSize() throws Exception {
+        LOG.info("ServiceConfigTests : testSmallMaxInboundMessageSize() : test very small MaxInboundMessageSize.");
+
+        setServerConfiguration(GrpcClientOnly, GRPCTARGET_MSGSIZESM);
+        GrpcClientOnly.waitForConfigUpdateInLogUsingMark(appName);
+        String contextRoot = "HelloWorldClient";
+        try (WebClient webClient = new WebClient()) {
+
+            // Construct the URL for the test
+            URL url = GrpcTestUtils.createHttpUrl(GrpcClientOnly, contextRoot, "grpcClient");
+            HtmlPage page = (HtmlPage) webClient.getPage(url);
+
+            // Log the page for debugging if necessary in the future.
+            Log.info(c, name.getMethodName(), page.asText());
+            Log.info(c, name.getMethodName(), page.asXml());
+
+            assertTrue("the servlet was not loaded correctly",
+                       page.asText().contains("gRPC helloworld client example"));
+
+            HtmlForm form = page.getFormByName("form1");
+
+            // set a name in the form, which we'll expect the RPC to return
+            HtmlTextInput inputText = (HtmlTextInput) form.getInputByName("user");
+            inputText.setValueAttribute("us3r1");
+
+            // set the port of the grpcserver in the form
+            HtmlTextInput inputPort = (HtmlTextInput) form.getInputByName("port");
+            inputPort.setValueAttribute(String.valueOf(GrpcServerOnly.getHttpDefaultPort()));
+
+            // set the hostname of the gprcserver in the form
+            HtmlTextInput inputHost = (HtmlTextInput) form.getInputByName("address");
+            inputHost.setValueAttribute(GrpcServerOnly.getHostname());
+
+            // submit to the grpcClient, and execute the RPC
+            HtmlSubmitInput submitButton = form.getInputByName("submit");
+            page = submitButton.click();
+
+            // Log the page for debugging if necessary in the future.
+            Log.info(c, name.getMethodName(), page.asText());
+            assertTrue("the gRPC request did not complete correctly", page.asText().contains("us3r1"));
```

Comment:
Going by the comment, I'd expect this test to check for an error case.  If you set a very small max inbound message size and request a response that's larger than that value, will it give an error?  This test case as it's written doesn't test that since `us3r1` is less than 16 bytes.

Reply:
Ooops, meant to circle back to that test after I got the others working and forgot.  I'll have it fixed up momentarily.